validator/project_validator_test.go

```diff
@@ -2437,6 +2437,65 @@ func TestCheckTaskCommands(t *testing.T) {
 	})
 }
 
+func TestValidateModuleUsageInGitGetProject(t *testing.T) {
+	for testName, testCase := range map[string]func(*testing.T, *model.Project){
+		"Valid git.get_project": func(t *testing.T, p *model.Project) {
+			errs := validateModuleUsageInGitGetProject(p)
+			assert.Len(t, errs, 0)
+		},
+		"Invalid git.get_project": func(t *testing.T, p *model.Project) {
+			// Add the task to a build variant that does not have all the modules needed.
+			p.BuildVariants[1].Tasks = append(p.BuildVariants[1].Tasks, model.BuildVariantTaskUnit{Name: "test"})
+			errs := validateModuleUsageInGitGetProject(p)
+			require.Len(t, errs, 1)
+			assert.Contains(t, errs[0].Message, "bar")
```

Comment:
Since we know what the error will be in full, I'd preferrer adding the whole error message 
`"build variant 'bv2' with task 'test' with 'git.get_project' command uses module/revision 'bar' that is not present in build variant"` or a bigger part of it. It makes it easier to read the test and future proof against changes.